frontend/src/sections/EventSection/Main.tsx

```diff
@@ -14,7 +14,7 @@ import { EventsResponse } from '@/types'
 
 const Main: React.FC = async () => {
   const { data: events } = await fetchData<EventsResponse>(
-    API_ENDPOINTS.allEvents,
+    API_ENDPOINTS.events,
```

Comment:
This will not work after changes done in `API_ENDPOINTS`. Same with `API_ENDPOINTS.newsletter`.

Reply:
surprisingly it is working, but to maintain consistency, I will update